pkgs/development/python-modules/froide/default.nix

```diff
@@ -0,0 +1,203 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+  gdal,
+  geos,
+  pnpm,
+  nodejs,
+  postgresql,
+  postgresqlTestHook,
+  playwright-driver,
+}:
+let
+
+  python = python3.override {
+    packageOverrides = self: super: {
+      django = super.django.override { withGdal = true; };
+
+      # Project discontinued upstream
+      # https://github.com/okfde/froide/issues/893
+      django-fsm = self.buildPythonPackage rec {
+        pname = "django-fsm";
+        version = "3.0.0";
+        pyproject = true;
+
+        src = fetchFromGitHub {
+          owner = "viewflow";
+          repo = "django-fsm";
+          rev = "refs/tags/${version}";
+          hash = "sha256-woN0F4hTaPk8HTGNT6zQlZDJ9SCVRut9maKSlDmalUE=";
+        };
+
+        build-system = [ self.setuptools ];
+
+        dependencies = [ self.django ];
+
+        checkInputs = [ self.django-guardian ];
+
+        checkPhase = ''
+          ${python3.interpreter} tests/manage.py test
+        '';
+
+        pythonImportsCheck = [ "django_fsm" ];
+
+        meta = {
+          description = "Django friendly finite state machine support";
+          homepage = "https://github.com/viewflow/django-fsm";
+          license = lib.licenses.mit;
+          knownVulnerabilities = [ "Package is marked as discontinued upstream." ];
+          maintainers = [ lib.maintainers.onny ];
+        };
+      };
```

Comment:
Not necessary — just add it as another regular Python module and drop it when upstream fixes it